lib/ansible/modules/system/java_cert.py

```diff
@@ -15,20 +15,25 @@
 module: java_cert
 version_added: '2.3'
 short_description: Uses keytool to import/remove key from java keystore(cacerts)
+extends_documentation_fragment: files
 description:
   - This is a wrapper module around keytool, which can be used to import/remove
     certificates from a given java keystore.
 options:
   cert_url:
     description:
-      - Basic URL to fetch SSL certificate from. One of cert_url or cert_path is required to load certificate.
+      - Basic URL to fetch SSL certificate from. One of cert_url or cert_path or cert_content is required to load certificate.
```

Comment:
Use [formatting functions](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation): `I(cert_url)`:
```suggestion
      - Basic URL to fetch SSL certificate from. One of I(cert_url) or I(cert_path) or I(cert_content) is required to load certificate.
```